lib/mini_profiler/config.rb

```diff
@@ -81,6 +80,11 @@ def merge!(config)
         end
       end
 
+      def set_positions!
+        position = (self.position && self.position.match("-")) ? self.position.split("-") : ["top", "left"]
+        instance_variable_set "@vertical_position", position.first
```

Comment:
this is weird, why use instance_variable_set? just set defaults in the constructor and then `@vertical_position =`